kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkTask.java

```diff
@@ -329,7 +340,10 @@ public void close(Collection<TopicPartition> partitions) {
   @Override
   public void stop() {
     try {
-      postCommitHook.close();
+      if (postCommitHook != null) {
+        postCommitHook.close();
+      }
```

Comment:
Not related to this PR, but in some cases `postCommitHook` is null and we get NPE.